### PR TITLE
To `polkadot-v0.9.25`

### PR DESCRIPTION
**Pull request by jiguantong**
Combine #106, #109.

### CODE REVIEW

File: runtime/crab-parachain/src/pallets/bridge_dispatch.rs [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 
-// --- paritytech ---
-use frame_support::traits::Everything;
 // --- darwinia-network ---
 use crate::*;
+use bp_message_dispatch::{Everything, IntoDispatchOrigin as IntoDispatchOriginT};
 use bp_messages::{LaneId, MessageNonce};
 use pallet_bridge_dispatch::Config;
 
+pub struct IntoDispatchOrigin;
```
Comment (acgxv) [inline runtime/crab-parachain/src/pallets/bridge_dispatch.rs:9]: Check

Comment (jiguantong) [reply runtime/crab-parachain/src/pallets/bridge_dispatch.rs:9]: @AsceticBear cc

File: runtime/crab-parachain/src/pallets/bridge_dispatch.rs [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+		frame_system::RawOrigin::Signed(id.clone()).into()
+	}
+}
+
 impl Config<WithCrabDispatch> for Runtime {
 	type AccountIdConverter = bp_crab_parachain::AccountIdConverter;
 	type BridgeMessageId = (LaneId, MessageNonce);
 	type Call = Call;
-	type CallFilter = Everything;
+	type CallValidator = Everything;
```
Comment (acgxv) [inline runtime/crab-parachain/src/pallets/bridge_dispatch.rs:20]: @HackFisher  @xiaoch05

File: runtime/crab-parachain/src/pallets/bridge_dispatch.rs (hunk 1 above)
Comment (boundless-forest) [reply runtime/crab-parachain/src/pallets/bridge_dispatch.rs:9]: LGTM

**Review by acgxv — APPROVED** [review]

**Pull request merged**